sound/soc/codecs/rt5682.c

```diff
@@ -2464,6 +2821,10 @@ static const struct regmap_config rt5682_regmap = {
 	.num_reg_defaults = ARRAY_SIZE(rt5682_reg),
 	.use_single_read = true,
 	.use_single_write = true,
+#if IS_ENABLED(CONFIG_SND_SOC_RT5682_SDW)
```

Comment:
not sure this is going to work either. This should be a regmap specific for SoundWire.